packages/rn-tester/RNTesterPods.xcodeproj/project.pbxproj

```diff
@@ -748,15 +768,15 @@
 			isa = XCBuildConfiguration;
 			baseConfigurationReference = 98233960D1D6A1977D1C7EAF /* Pods-RNTester.debug.xcconfig */;
 			buildSettings = {
-				ALWAYS_EMBED_SWIFT_STANDARD_LIBRARIES = YES;
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
-				CLANG_CXX_LANGUAGE_STANDARD = "c++14";
```

Comment:
Question: why did we need to remove `CLANG_CXX_LANGUAGE_STANDARD`?

Reply:
Oh I should have mentioned this, there was a warning in cocoapods when running pod install because it cannot override this value. I verified the value that is inherited is also c++14.